themes-default/slim/static/js/templates/add-recommended.vue

```diff
@@ -1,6 +1,5 @@
 <template>
     <div>
```

Comment:
Now you don't need that wrapper div :)

Reply:
Didn't want to remove them incase we had any styles relying on the div depth. Haven't had a chance to check.

Reply:
I did, it's okay, can be removed.
Keep the `<div id="addShowPortal">` though.